templates/go/events/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+builders:
+  default: paketobuildpacks/builder:base
+  base: paketobuildpacks/builder:base
+  full: paketobuildpacks/builder:full
+
+buildpacks:
+  - paketo-buildpacks/go-dist
+  - docker.io/lanceball/boson-go # github.com/lance/buildpacks
+
+healthEndpoints:
+  liveness: /health/liveness
+  readiness: /health/readiness
```

Comment:
If these defaults were instead defined as constants `DefaultLiveness` and `DefaultReadiness` in `deployer.go`, they could be omitted from the default embedded templates.. and from any Function project for that matter, unless the developers (of either Function or Language Pack) specifically need to override.

Reply:
The only issue with this is that it does not allow a language pack provider to forego supplying these endpoints. It places a requirement on all language packs to either provide these or simply specify `/` as the health check endpoint.

Reply:
See the commit message where this is addressed. I suggest we document that language packs wishing to avoid implementing these health checks at `/health/readiness` and `/health/liveness` could use the root path with query parameters. E.g. `/?health=readiness`.